docs/source/quicktour.mdx

```diff
@@ -14,186 +14,186 @@ specific language governing permissions and limitations under the License.
+
+Let's return to our example and see how we can use the `AutoClasses` to replicate the results of the [`pipeline`].
+
+## Using the AutoTokenizer
 
-```py
+A tokenizer is responsible for preprocessing text in a way that is understandable by the model. First, the tokenizer will split a given text into words called *tokens*. There are multiple rules that govern the tokenization process, including how to split a word and at what level (learn more [here](./tokenizer_summary)). The most important thing to remember though is you need to instantiate the tokenizer with the name of the model to ensure you're using the same tokenization rules a model was pretrained with.
+
+Load a tokenizer with [`AutoTokenizer`]:
+
+```python
 >>> from transformers import AutoTokenizer, AutoModelForSequenceClassification
->>> model_name = "distilbert-base-uncased-finetuned-sst-2-english"
->>> pt_model = AutoModelForSequenceClassification.from_pretrained(model_name)
+>>> model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
 >>> tokenizer = AutoTokenizer.from_pretrained(model_name)
 ===PT-TF-SPLIT===
 >>> from transformers import AutoTokenizer, TFAutoModelForSequenceClassification
->>> model_name = "distilbert-base-uncased-finetuned-sst-2-english"
->>> tf_model = TFAutoModelForSequenceClassification.from_pretrained(model_name)
+>>> model_name = "nlptown/bert-base-multilingual-uncased-sentiment"
 >>> tokenizer = AutoTokenizer.from_pretrained(model_name)
 ```
 
-### Using the tokenizer
-
-We mentioned the tokenizer is responsible for the preprocessing of your texts. First, it will split a given text in
-words (or part of words, punctuation symbols, etc.) usually called _tokens_. There are multiple rules that can govern
-that process (you can learn more about them in the [tokenizer summary](tokenizer_summary)), which is why we need
-to instantiate the tokenizer using the name of the model, to make sure we use the same rules as when the model was
-pretrained.
+Next, the tokenizer converts the tokens into numbers in order to construct a tensor as input to the model. This is known as the model's *vocabulary*.
 
-The second step is to convert those _tokens_ into numbers, to be able to build a tensor out of them and feed them to
-the model. To do this, the tokenizer has a _vocab_, which is the part we download when we instantiate it with the
-`from_pretrained` method, since we need to use the same _vocab_ as when the model was pretrained.
+Pass your text to the tokenizer:
 
-To apply these steps on a given text, we can just feed it to our tokenizer:
-
-```py
->>> inputs = tokenizer("We are very happy to show you the 🤗 Transformers library.")
+```python
+>>> input = tokenizer("We are very happy to show you the 🤗 Transformers library.")
```

Comment:
`input` is a reserved word in Python so we should avoid using it. How about `encoding`?